frontend/src/components/Back.js

```diff
@@ -0,0 +1,16 @@
+import Button from "./Button";
+import React, {Component} from 'react';
+import {Link} from "react-router-dom";
+
+
+class Back extends Component {
+    render() {
+        // Strip the last two parts of the url (which are in format `/view/id` to go `up` in the dashboard's structure.
+        const path = this.props.path.split("/").slice(0, -2).join("/") || "/";
+        return (
+            <Link to={path}><Button>Go back</Button></Link>
+        );
+    }
+}
+
+export default Back;
```

Comment:
Alternatively, since this is just the browser history, you can just create regular links to the parent page, i.e. the per-user page can link to the board page, the board page to the board selection. Encoding this logic in the button itself is a bit brittle since that can change with a minor change to the app.

Reply:
+1 to the second idea! Thanks!
The first one wouldn't work if we opened the specific board directly via the URL.